gapii/interceptor-lib/cc/lib/ARM/target_arm.cc

```diff
@@ -114,6 +114,7 @@ static void *calculatePcRelativeAddressArm(void *data, size_t pc_offset,
 
   data_addr += pc_offset;  // Add the PC
   data_addr += 8;          // Add the 8 byte implicit offset
+  data_addr += offset;     // Add the offset
```

Comment:
This looks like a nasty bug you've fixed. Was nothing passing `offset` before?

Reply:
Only ARM::Bcc seems to have been affected (non-Thumb).
The Thumb calculation is correct.